portfolio/src/main/java/com/google/sps/servlets/NameServlet.java

```diff
@@ -104,4 +111,14 @@ public static String getLastName(String id) {
     String lastName = (String) entity.getProperty("lastName");
     return lastName;
   }
+
+  // returns whether a user has voted
+  public static String getHasVoted(String id) {
```

Comment:
Why not make this a boolean so the output is more clear? A string can have many different values.